src/modbus-tcp-client.js

```diff
@@ -10,6 +10,7 @@ module.exports = stampit()
             currentRequestId    = reqId,
             closedOnPurpose     = false,
             reconnect           = false,
+            buffer              = new Buffer(0),
```

Comment:
`new Buffer` is officially deprecated in favor of `Buffer.from` or `Buffer.alloc` however this initialization is not needed anyway here, right?